lascli/parser/workflows.py

```diff
@@ -61,7 +74,9 @@ def create_workflows_parser(subparsers):
     create_workflow_parser.add_argument('specification_path')
     create_workflow_parser.add_argument('--name')
     create_workflow_parser.add_argument('--description')
-    create_workflow_parser.add_argument('--error-config', type=str)
+    create_workflow_parser.add_argument('--error-config', help='path to the error configuration for the workflow')
+    create_workflow_parser.add_argument(
+        '--completed-config', help='path to the configuration for the job that runs on completion of an execution')
```

Comment:
Is it obvious what a 'job' is in this context? Maybe just keep it simple unless we can fully explain what it entails. "path to the execution completed configuration for the workflow"